akka-sample-sharding-scala/README.md

```diff
@@ -132,6 +134,32 @@ Each node's log will show its dynamic weather port opened for weather stations t
 [2019-11-04 14:43:45,861] [INFO] [akka.actor.typed.ActorSystem] [KillrWeather-akka.actor.default-dispatcher-16] [] - WeatherServer online at http://127.0.0.1:8081/
 ```
 
+### Interacting with the HTTP endpoint manually
+
+With the cluster running you can interact with the HTTP endpoint using `curl`
+
+List known stations on a node 
+```
+curl http://localhost:12553/weather
+```
+    
+Add a station 123 on a node: 
+```
+curl -XPOST http://localhost:12553/weather/123
+```
```

Comment:
The station registry (previously called Guardian, which is confusing since there is a User Guardian in Akka, so renamed to `Stations` and introduced an actual `Guardian` actor bootstrapping the app) runs on each node and has a local registry, this is strange in a sample showcasing a distributed system, I wonder if we should rather throw that part away completely?

Reply:
Throw the app's guardian out?

Reply:
The old `Guardian` that I renamed to `Stations`. After sleeping on it I think we should just drop that as it doesn't make sense to have a node local set of station ids where the actual stations are then created through sharding.